UltiSnips/javascript.snippets

```diff
@@ -56,23 +56,26 @@ ${1:class_name}.prototype.${2:method_name} = function`!p snip.rv = space_before_
 
 endsnippet
 
-snippet fun "function (fun)" w
+snippet fun "function (named)" b
 function ${1:function_name}`!p snip.rv = space_before_function_paren(snip)`(${2:argument}) {
 	${VISUAL}$0
 }
 endsnippet
 
-snippet vf "Function assigned to var"
+snippet =fun "function (assigned to var)"
```

Comment:
Could you put back the original trigger please? People will be using this.

Reply:
Sure, but what about anf vs fun for anonymous functions? Is that change okay? (If not, then I don't think this change has much left to offer.)

Reply:
We should always aim to keep the existing triggers as people may be using them. We can add new ones :)

Reply:
I won't add `=fun` and I'll keep `anf` untouched (as a copy).